src/core/Table/Table.tsx

```diff
@@ -562,9 +561,10 @@ export const Table = <
                 onBottomReached={onBottomReachedRef}
                 intersectionMargin={intersectionMargin}
                 state={state}
-                key={rowProps.key}
+                key={row.getRowProps().key}
```

Comment:
It is really a lightweight function, so I can call it multiple times:
![image](https://user-images.githubusercontent.com/36186912/127610528-3c041106-0d61-472b-a6e7-fd582b5462be.png)